libs/langchain/langchain/llms/azureml_endpoint.py

```diff
@@ -92,44 +115,70 @@ def format_response_payload(self, output: bytes) -> str:
         """
 
 
-class OSSContentFormatter(ContentFormatterBase):
+class GPT2ContentFormatter(ContentFormatterBase):
```

Comment:
do we need to change name? main concern being this is a breaking change

Reply:
The name is more specific and correct than the previous. Might it be better to deprecate `OSSContentFormatter` instead and introduce `GPT2ContentFormatter`?

Reply:
I reinstated `OSSContentFormatter` and it uses the logic from `GPT2ContentFormatter`. This should not cause any breaking changes now, since they both encompass the same logic, but the latter is a more proper name than the former. A warning is also issued telling the user to use `GPT2ContentFormatter`.